azure-pipelines-arm.yml

```diff
@@ -17,6 +17,7 @@ steps:
     node script/download-circleci-artifacts.js --buildNum=$CIRCLE_BUILD_NUM --name=dist.zip --dest=$ZIP_DEST
     cd $ZIP_DEST
     unzip -o dist.zip
+    xattr -cr Electron.app    
```

Comment:
@jkleinsc What is the reasoning behind clearing the extended attributes here?

Reply:
@jviotti just to make sure the quarantine attribute is removed from the downloaded file.